### PR TITLE
Feature/hw 3

### PR DESCRIPTION
**Pull request by lototska**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,46 +4,58 @@
...
         <title>Document</title>
     </head>
-    <body>
-        <ul>
-            <li><a href="#yak1">lifestyle</a></li>
-            <li><a href="#yak2">photodiary</a></li>
-            <li><a href="#yak3">music</a></li>
-            <li><a href="#yak4">travel</a></li>
+    <body class="container">
+        <ul class="sate-navigation">
```
Comment (yaroslav-lototskyi) [inline index.html]: wrap to tag nav

File: index.html [hunk 2]
```diff
@@ -4,46 +4,58 @@
...
+            <li><a href="#">lifestyle</a></li>
+            <li><a href="#">photodiary</a></li>
+            <li><a href="#">music</a></li>
+            <li><a href="#">travel</a></li>
         </ul>
-        <ul style="list-style: none">
```
Comment (yaroslav-lototskyi) [inline index.html:16]: wrap to tag article or section

File: index.html [hunk 3]
```diff
@@ -4,46 +4,58 @@
...
+                <img class="list_item_img" src="img/image6.jpg" width="420" height="274" alt="road">
+                <p class="photo-description">Lifestyle</p>
+                <h2>Cold winter days</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris </p>
             </li>
         </ul>
-        <button>load more</button>
+        <button class="button">Lored more</button>
     </body>
 </html>
```
Comment (yaroslav-lototskyi) [inline index.html]: add empty line

File: styles/style.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    font-family: "Ubuntu";
+    color: #b4ad9e;
+}
+.sate-navigation {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    margin-bottom: 120px;
+    margin-top: 62px;
```
Comment (yaroslav-lototskyi) [inline styles/style.css]: in this scope you have 3 properties with```margin```. why ? replace on one tag ```margin```

File: styles/style.css [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    margin-right: 110px;
+    font-size: 14px;
+}
+.sate-navigation li:last-child {
+    margin-right: 0;
+}
+.list { 
+    list-style: none;
+    margin: 0;
+    margin-bottom: 54px;
```
Comment (yaroslav-lototskyi) [inline styles/style.css]: tha same with tag margin

File: styles/style.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+}
+.photo-description {
+    margin: 0;
+    margin-bottom: 30px;
+}
+.text {
+    margin: 0;
+}
+.list p {
+font-size: 14px;
```
Comment (yaroslav-lototskyi) [inline styles/style.css]: align

File: styles/style.css [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+   border: 1px solid #626262; 
+   background-color: #fff;
+   cursor: pointer;
+   color: #626262;
+   font-size: 20px;
+   margin-bottom: 70px;
+   text-align: center;
+   margin-left: 366px;
+   margin-right: 368px;
+}
```
Comment (yaroslav-lototskyi) [inline styles/style.css:95]: empty line

File: styles/style.css [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+   height: 62px;
+   border: 1px solid #626262; 
+   background-color: #fff;
+   cursor: pointer;
+   color: #626262;
+   font-size: 20px;
+   margin-bottom: 70px;
+   text-align: center;
+   margin-left: 366px;
+   margin-right: 368px;
```
Comment (yaroslav-lototskyi) [inline styles/style.css]: tha same with tag margin

**Comment by lototska** [comment]
Homework 3 finish
https://github.com/lototska/hw-html/pull/3 code sourse
https://lototska.github.io/hw-html/ git page

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Без
вирусов. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#m_-5713271888084712583_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

чт, 22 квіт. 2021 о 17:31 Yaroslav Lototskyi ***@***.***>
пише:

**Pull request merged**